server/views/pages/bookedtoday/summaryMoveOnly.njk

```diff
@@ -85,7 +85,7 @@
                             src="/assets/images/placeholder-image.png"
                             class="govuk-!-margin-bottom-8"
                             width="160px"
-                            alt="Headshot of {{ arrival.lastName }}, {{ arrival.firstName }}"
+                            alt="No photo available"
```

Comment:
The default for the img  tag is set to the `placeholder image` therefore I have just updated the `alt` text